woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-subscriptions.php

```diff
@@ -884,11 +896,13 @@ public function admin_validate_payment_meta( $meta ) {
 
 		// payment token
 		if ( empty( $meta['post_meta'][ $prefix . 'payment_token' ]['value'] ) ) {
+			/* translators: Context: Error message. Placeholder: %s - Required value not found or not as expected */
```

Comment:
This seems confusing to me. How about:

```suggestion
			/* translators: Context: Error message. Placeholder: %s - Label of the related value */
```